pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -253,16 +266,21 @@ func (r *reconcilerBase) createOrPatchDeployment(ctx context.Context, declared *
 // This function aims to address the fight among Autopilot, VPA and the reconciler since they all update the resources.
 // Below is the resolution:
 // 1. If VPA is enabled (no matter if it is a standard cluster or an Autopilot cluster),
```

Comment:
Need to update all comments about "VPA enabled". We didn't check whether VPA is enabled in the cluster or not. We're only checking whether the --allow-vertical-scale flag is enabled in the reconciler manager.

Reply:
update comments